sim/druid/item_sets_pve.go

```diff
@@ -436,6 +438,9 @@ var ItemSetCunningOfStormrage = core.NewItemSet(core.ItemSet{
 			druid.RegisterAura(core.Aura{
 				Label: "S03 - Item - T2- Druid - Feral 4P Bonus",
 				OnInit: func(aura *core.Aura, sim *core.Simulation) {
+					if druid.form != Cat {
+						return
+					}
```

Comment:
Since forms can be changed mid-combat I don't think we want to add this restriction